Source/WebCore/css/CSSPseudoSelectors.json

```
@@ -514,6 +514,11 @@
             "conditional": "ENABLE(VIDEO)",
             "user-agent-part-string": true
         },
+        "cue-background": {
```

If this is only used in UA stylesheet, I would make this change:

```suggestion
        "-internal-cue-background": {
```

Otherwise the web can use it and do feature detection/hacks that can cause web compat problems later on